README.md

```diff
@@ -3,6 +3,30 @@ PowerAcademyPOO
 
 1.Write a function : ```public int max(int[] vec)```  . It receives a vector of numbers and returs the highest number.
 
+#include <iostream>
+#include <vector>
+using namespace std;
+
+
+   int Max(int vec[], int n) 
+{ 
+    
+    if (n == 1) 
+        return vec[0]; 
+    return max(vec[n-1], Max(vec, n-1)); 
+    
+} 
+  
+
+int main() 
+{ 
+    int vec[] = {1, 2, 3, 4, 5}; 
+    int n = sizeof(vec)/sizeof(vec[0]); 
+    cout <<  Max(vec, n); 
+    return 0; 
+
+} 
+
 2.Write a function : ```public int[] merge(int[] v1, int[] v2)``` . It received 2 vectors of numbers sorted in acsending order and should return a vector with all numbers from v1 and v2 sorted in ascending order
 
```

Comment:
2.

#include <iostream>

#include <vector>

#include <bits/stdc++.h>

using namespace std;




void merge(int v1[], int v2[], int n1, int n2, int v3[]) 

{ 
    int i = 0, j = 0, k = 0; 
  
    
   
 while (i<n1 && j <n2) 
  
  {  
        if (v1[i] < v2[j]) 
       
     v3[k++] =v1[i++]; 
       
 else
        
   v3[k++] = v2[j++]; 
    } 
  
    

    while (i < n1) 
      
  v3[k++] = v1[i++]; 
  
    
 
   while (j < n2) 
      
  v3[k++] =v2[j++]; 
} 
  

 
int main() 

{ 
   
 int v1[] = {1, 3, 5, 7}; 
 
   int n1 = sizeof(v1) / sizeof(v1[0]); 
  
   
 int v2[] = {2, 4, 6, 8}; 
  
 int n2 = sizeof(v2) / sizeof(v2[0]); 
  
    int v3[n1+n2];
 
    merge(v1, v2, n1, n2, v3); 
  
 
   cout << "dupa merge" <<endl; 
  
  for (int i=0; i < n1+n2; i++) 
      
  cout << v3[i] << " "; 
  
  
  return 0; 
}